pkg/utils/cluster.go

```diff
@@ -42,6 +42,7 @@ func k8sSingleNodeClusterStatus(c client.Client) (bool, error) {
 
 func openshiftSingleNodeClusterStatus(c client.Client) (bool, error) {
 	infra := &configv1.Infrastructure{}
+	configv1.AddToScheme(c.Scheme())
```

Comment:
Wondering if this should be added here: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/main.go#L57

Reply:
I wasn't sure either. I figured since this is a code path for Openshift clusters only it would be best to keep it close to the one spot it's being used, for now at least.

Reply:
Maybe we can keep it here and move it later if needed?
@SchSeba wdyt?

Reply:
I think is better to move this to the main.go file to have all the scheme changes/configuration in one please

Reply:
Sounds good, moved it to `main.go`